admin/js/boldgrid-backup-admin.js

```diff
@@ -112,14 +114,77 @@ BoldGrid.Backup = function( $ ) {
 		} );
 	};
 
+	/**
+	 * Auto Update Themes
+	 *
+	 * Add functionality for WP >= 5.5 theme update links.\
+	 *
+	 * @since SINCEVERSION
+	 */
+	self.autoUpdateThemes = function( link ) {
+		var data = {
+			action: 'boldgrid_backup_auto_update',
+			_wpnonce: BoldGridBackupAdmin.auto_update_nonce,
+			data: link.dataset
+		};
+		$.post( ajaxurl, data, function( response ) {
+			if ( JSON.parse( response ).settings_updated ) {
+				location.reload();
+			}
+		} );
+	};
+
+	/**
+	 * Auto Update Links
+	 *
+	 * Adds functionality for WP >= 5.5 update links.
+	 *
+	 * @since SINCEVERSION
+	 */
+	self.autoUpdateLinks = function() {
+		$( '.boldgrid-backup-enable-auto-update' ).click( function( e ) {
+			var data = {
+				action: 'boldgrid_backup_auto_update',
+				_wpnonce: BoldGridBackupAdmin.auto_update_nonce,
+				data: this.dataset
+			};
+			e.preventDefault();
+			$.post( ajaxurl, data, function( response ) {
+				if ( JSON.parse( response ).settings_updated ) {
+					location.reload();
+				}
+			} );
+		} );
+	};
+
 	$( function() {
+		var noDivider;
 		self.bindHelpClick();
 		self.hideBackupNotice();
 		self.updatePremiumLink();
 
+		// fire function for auto update links.
+		self.autoUpdateLinks();
+
 		$( 'body' ).on( 'click', '[data-bgbkup-toggle-target]', self.onClickToggle );
 		$( 'body' ).on( 'make_notices_dismissible', self.makeNoticesDismissible );
 
+		/*
```

Comment:
If we're going to remove functionality, we need to discuss. There's no way to get this working? Can you point me to the WordPress code that is having a problem with TU?